Content.Shared/MedicalScanner/SharedMedicalScannerComponent.cs

```diff
@@ -16,25 +16,29 @@ public abstract class SharedMedicalScannerComponent : Component, IDragDropOn
         public class MedicalScannerBoundUserInterfaceState : BoundUserInterfaceState
         {
             public readonly EntityUid? Entity;
-            public readonly Dictionary<DamageGroupPrototype, int> DamageGroup;
-            public readonly Dictionary<DamageTypePrototype, int> DamageTypes;
+            public readonly Dictionary<string, int> DamagePerSupportedGroupID;
+            public readonly Dictionary<string, int> DamagePerTypeID;
             public readonly bool IsScanned;
 
+            // TODO QUESTION DrSmugleaf previously commented here saying that these dicitionaries should be using
```

Comment:
Please someone who knows UI or networking check this. I was just doing whatever looked right and got working results.

Reply:
Seems fine